docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     environment:
       - BUILD=false # In case you want to re-generate a all the images, not recommended
       - CLEAN=true # it remove the images directory
-      - UNATTENDED=false # UNATTENDED version
+      - UNATTENDED=true # UNATTENDED version
```

Comment:
revert personal changes 

again
This is/was a personal test branch and i have been testing it offline and saved my local work on my branch and pushed it and then when went back to github i found that @dappnode/core had started to move on it themselves so my last commit from my VS-Code wasnt made as a PR because I didnt know anyone was planning on implementing it just yet for everyone especially.
Reverting this one now